src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -3,50 +3,112 @@
 /**
  * <p>Реалізувати свій ArrayList який імплементує інтерфейс List</p>
  */
+
+import java.util.Arrays;
+
 public class ArrayList<T> implements List<T> {
+    private Object[] array = new Object[16];
```

Comment:
why 16?

better rename to `elements` or `dataElements`

Reply:
You said me at retrospective that 8 and 16 are good number for arrays :-) 
I don't have any "elements". What did you mean?